master/getting-started/kubernetes/installation/hosted/k8s-backend/index.md

```diff
@@ -31,6 +31,8 @@ To install Calico, ensure you have a cluster which meets the above requirements
 kubectl apply -f {{site.url}}/{{page.version}}/getting-started/kubernetes/installation/hosted/k8s-backend/calico.yaml
 ```
 
+>[Click here to view the above yaml directly.](calico.yaml)
```

Comment:
When I try this out, clicking on the link downloads the yaml rather than opening in a new tab.  Is that the desired behavior?